test/app.js

```diff
@@ -18,21 +18,18 @@ const errConst = require('../src/error_const');
 
 const createTestApp = h.createTestApp;
 const appInfo = h.appInfo;
-const createTestAppNoInit = h.createTestAppNoInit;
 const createAuthenticatedTestApp = h.createAuthenticatedTestApp;
-const createTestAppWithNetworkCB = h.createTestAppWithNetworkCB;
-const createTestAppWithOptions = h.createTestAppWithOptions;
 const { autoref } = require('../src/helpers');
 
 describe('Smoke test', () => {
   it('should return undefined value if log option is true, however app logging is not initialised', async () => {
-    const app = await createTestAppNoInit(null, { log: true });
+    const app = autoref(new App(h.appInfo, null, { log: true }));
```

Comment:
Can't we use `createTestApp` here too?

Reply:
Here I need the app to be created without calling `app.init()`, as it is in [createTestApp](https://github.com/hunterlester/safe_app_nodejs/blob/c68e55a1d18525e7213b80643523508f50c289ee/test/helpers.js#L38).
This was the only test where I was using `createTestAppNoInit`.